sktime/dists_kernels/tests/test_all_dist_kernels.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pytest
+
+from sktime.dists_kernels._base import BasePairwiseTransformer
+from sktime.utils import all_estimators
+from sktime.utils._testing.panel import make_transformer_problem
+from sktime.tests._config import ESTIMATOR_TEST_PARAMS
+
+PAIRWISE_TRANSFORMERS_TAB = all_estimators(
```

Comment:
Minor comment:
I notice that `_TAB` has been included here, which is contrary to the conventions used in _base

Reply:
@chrisholder I see you have updated tests to remove the _TAB. I was actually hoping the conventions in the tests would be adopted in base! 😅  But being consistent is better in my mind than having different conventions.

Reply:
hm, I don't think it would be a big issue either way. We just need to be consistent, which it now is.